app/ui-message/client/messageBox/messageBoxAudioMessage.js

```diff
@@ -152,7 +150,6 @@ Template.messageBoxAudioMessage.events({
 			recordingRoomId.set(this.rid);
 		} catch (error) {
 			instance.state.set(null);
-			instance.isMicrophoneDenied.set(true);
```

Comment:
what will happen if the mic permission was assumed to be granted and it fails when clicked? since this line was removed I assume it will silently fail, is that right?

Reply:
It will keep the button, which is useful when the permission is accidentally denied; the next click will retrigger the permission dialog.